travvy/src/components/LogIn.vue

```diff
@@ -0,0 +1,97 @@
+
+<template>  
+
+  <div class="LogIn">
+        <h2>Log In to TRAVVY</h2>
+        <form>
+        <!-- this form takes the sign up input from users - binds to the user form --> 
+          <label>Email: </label>
+          <input type="text" v-bind="UserForm"/> 
+          <br>
+          <br>
+          <label>Password: </label>
+          <input type="password" v-bind="UserForm"/> 
+        </form>
+        <br>
+        <br>
+        <button class="search"><span>Log In</span></button>
```

Comment:
Once the backend is created - it will be interesting to see the method behind the log in button. Good job!